cloudclient/options.go

```diff
@@ -20,6 +22,8 @@ const (
 	authorizationHeader           = "Authorization"
 	authorizationBearer           = "Bearer"
 	temporalCloudAPIVersionHeader = "temporal-cloud-api-version"
+
+	sdkVersion = "0.2.0"
```

Comment:
So we need to keep updating this every time we cut a release?
Anyway to automate this?

I think we should update our runbook with this manual step.

Reply:
https://github.com/temporalio/sdk-go/blob/master/internal/version.go#L35
Server go sdk does this too.

Reply:
Yeah I don't know of a good way to automate this for libraries. For compiled software there's a nicer solution.